docs/how_to_setup_object_storage.md

```diff
@@ -39,6 +73,100 @@ $ ./juicefs format \
     localhost test
 ```
 
+## Google Cloud Storage
+
+Cause Google Cloud doesn't have access key and secret key, the `--access-key` and `--secret-key` options can be omitted. Please follow Google Cloud document to know how [authentication](https://cloud.google.com/docs/authentication) and [authorization](https://cloud.google.com/iam/docs/overview) work. Typically, when you running inside Google Cloud, you already have permission to access the storage.
+
+And because bucket name is [globally unique](https://cloud.google.com/storage/docs/naming-buckets#considerations), when you specify the `--bucket` option could just provide its name. For example:
+
+```bash
+$ ./juicefs format \
+    --storage gs \
+    --bucket gs://<bucket> \
+    ... \
+    localhost test
+```
+
+## Azure Blob Storage
+
+Besides provide authorization information through `--access-key` and `--secret-key` options, you could also create a [connection string](https://docs.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string) and set `AZURE_STORAGE_CONNECTION_STRING` environment variable.
+
+## Backblaze B2 Cloud Storage
+
+You need first creating [application key](https://www.backblaze.com/b2/docs/application_keys.html). The "Application Key ID" and "Application Key" are the equivalent of access key and secret key respectively.
+
+## IBM Cloud Object Storage
+
+You need first creating [API key](https://cloud.ibm.com/docs/account?topic=account-manapikey) and retrieving [instance ID](https://cloud.ibm.com/docs/key-protect?topic=key-protect-retrieve-instance-ID). The "API key" and "instance ID" are the equivalent of access key and secret key respectively.
+
+## Scaleway Object Storage
+
+Please follow [this document](https://www.scaleway.com/en/docs/generate-api-keys) to learn how to get access key and secret key.
+
+## DigitalOcean Spaces Object Storage
+
+Please follow [this document](https://www.digitalocean.com/community/tutorials/how-to-create-a-digitalocean-space-and-api-key) to learn how to get access key and secret key.
+
+## Wasabi Cloud Object Storage
+
+Please follow [this document](https://wasabi-support.zendesk.com/hc/en-us/articles/360019677192-Creating-a-Root-Access-Key-and-Secret-Key) to learn how to get access key and secret key.
+
+## Alibaba Cloud Object Storage Service
+
+Please follow [this document](https://help.aliyun.com/document_detail/38738.html) to learn how to get access key and secret key.
```

Comment:
We should have these things covered: We support RAM for OSS, and the region and endpoint is optional for existing bucket. OSS has different endpoint depends on the network.